database/factories/UserFactory.php

```diff
@@ -15,14 +16,15 @@ class UserFactory extends Factory
     public function definition()
     {
         return [
-            'username' => $this->faker->name(),
-            'first_name' => $this->faker->name(),
-            'last_name' => $this->faker->name(),
-            'role_id' => 1,
-            'email' => $this->faker->unique()->safeEmail(),
+            'first_name' => $this->faker->firstName,
+            'last_name' => $this->faker->lastName,
+            'username' => $this->faker->userName,
+            'email' => $this->faker->unique()->safeEmail,
             'email_verified_at' => now(),
-            'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
+            'password' => bcrypt('12345678'),
             'remember_token' => Str::random(10),
+            'is_active' => true,
+            'role_id' => Role::all()->random()->id,
```

Comment:
Tại vì Role được factory sau User (trong DatabaseSeeder ông viết như thế) nên là nó sẽ báo lỗi này: InvalidArgumentException: You requested 1 items, but there are only 0 items available.
Nên là cho Role seed trước